### DISCUSSION TITLE
Monitor and control don't work on a 64-bit client

### DISCUSSION THREAD

**Discussion by androidepi** [epoptes/epoptes#207] [Q&A] [answered]
I'm using LTSP server on Ubuntu 22.04. Everything has been updated to the latest.

I've tried installing other viewer alternatives as per documentation. Same result.

Strangely, monitor and control work just fine on a 32-bit client with all the viewers I've tried.

Could it be related to Epoptes? Any ideas?

**Comment by alkisg** [comment, marked as answer]
Are you using Wayland? If so, switch to Xorg.
Wayland isn't very font of VNC viewers currently...

**Reply by androidepi** [reply]
Perfect. Switching to Xorg made the trick. Thank you.
